### ISSUE TITLE
Cannot change models

### ISSUE THREAD

**Issue by sharonbk** [SpheroidUniverse/SpheroidScript#4]
I changed the files to different models , after I changed their names to coin_gold.glb  and coin_silver.glb  and the app crushed and keep crushing

**Comment by AnthonySouls** [comment]
In your case, are the models being crushed when testing the app on the XR Hub or when you try to view them on Demiurge?

**Comment by sharonbk** [comment]
When I try to see my app on XR Hub, the app crush. When I try to see the models on the hub, it's showing that it's loading,  showing a ✔ sign and don't show the model. The models was designed with tilt brush,  could that be the reason?

**Comment by sharonbk** [comment]
Did you succeed changing models?

**Comment by AnthonySouls** [comment]
No. I am limited, I cannot test my apps on the XR Hub because my smartphone does not support ARCore.

**Comment by AnthonySouls** [comment]
![Suz](https://user-images.githubusercontent.com/41519295/89971073-a3086a00-dc30-11ea-8ec2-4aa9a40daa40.PNG)

I replicated the app with this model but I can't test it.

**Comment by AnthonySouls** [comment]
![test](https://user-images.githubusercontent.com/41519295/89971491-bc5de600-dc31-11ea-8460-1e9e9ee9c76f.PNG)

I saw that in the Layers tab we have the option to add testers but I still don’t know it’s working.

**Comment by sharonbk** [comment]
Do you want my email to add as a tester?

**Comment by AnthonySouls** [comment]
Yes, we can test to see if it works

**Comment by AnthonySouls** [comment]
I'm in the Telegram group, so don't put your email here for everyone to see send me in private. Telegram: @ARHorizon

**Comment by bezvozni** [comment]
did you succeed?

**Comment by sharonbk** [comment]
Nope

**Comment by SpheroidUniverse** [comment]
Thank you for submitting the issue! Can you please [provide us with your app id](https://github.com/SpheroidUniverse/SpheroidScript/blob/master/docs/submit-an-issue.md#tell-us-your-app-id) and [share access to your app](https://github.com/SpheroidUniverse/SpheroidScript/blob/master/docs/submit-an-issue.md#share-access-to-your-app-to-the-support-team) to our support team? This will help us to solve your issue. @sharonbk

**Comment by sharonbk** [comment]
I added you. The app called First AR Test
I already changed back the models to the original ones. 
I made a few changes in the code to see how I change animations, pretty nice. To bad that I don't see what changes do I made in the IDE , some kind of history because I change many things and can't remember what I changed and what is the original value.
Thank you

**Comment by hodlinn** [comment]
@sharonbk and @AnthonySouls I tried it and mine work perfectly. You can contact me on Telegram @augmentedAR so I can add you guys as tester or collaborators
